shim.h

```diff
@@ -148,7 +148,6 @@
 #include "include/tpm.h"
 #include "include/ucs2.h"
 #include "include/variables.h"
-#include "include/sbat.h"
```

Comment:
I don't understand this change.

Reply:
It's also included on line 142, I just dropped the dup. You may or may not have already fixed this in your branch.

Reply:
That's already fixed in your current tip and was dropped by my rebase.